pkg/manager/manager.go

```diff
@@ -101,8 +95,6 @@ func StartServer(httpServerAddress, configFilePath, certDirectory, caCertFile st
 			for _, value := range controllers.VerifierMap {
```

Comment:
Are the comments in the 2 lines above still applicable?

Reply:
It is especially important to update those comments. Updated accordingly.